### ISSUE TITLE
Where will the video be saved?

### ISSUE THREAD

**Issue by genkihoshino** [stefanopini/simple-HigherHRNet#16]
Hi,Thanks for the great code.
I am running it on GoogleColaboratory.
When I run it with a saved video, it only saves a still image called 'frame.png'.
Where is the video with heatmap saved?
If you need to write any additional code, I would be glad to know.
Thank you in advance for your help.

**Comment by stefanopini** [comment]
Hi, I'm happy you solved the issue! Could you post your solution for other people that may experience the same error?

The `live-demo.py` script should save a video named `output.avi` in the project main folder. It saves the video frames with the detected keypoints and skeletons, it doesn't save the predicted heatmap as far as I remember.

**Comment by genkihoshino** [comment]
Hi, thank you for taking a look at my question.
My mistake was that I did not install vlc and did not fill in the --save_video argument.
So I did `!apt install vlc` in GoogleColaboratory and did `!python scripts/live-demo.py --filename video.mp4 --save_video`. Then I was able to get `output.avi` in the simple-HigherHRNet folder.
Thanks to you, I was able to proceed with my HENet study. Thank you so much for your help.
